src/main/java/info/metadude/java/library/overpass/models/query/enums/Filter.java

```diff
@@ -0,0 +1,19 @@
+package info.metadude.java.library.overpass.models.query.enums;
+
+/**
+ * Tag request clauses (or "tag filters")
+ */
+public enum Filter {
```

Comment:
For readability reasons I suggest adding `_` in the enum values such as `NOT_EQUAL` instead of `NOTEQUAL`.